### PR TITLE
icu4c 68.2

### PR DESCRIPTION
**Pull request by miccal**
Continuation of https://github.com/Homebrew/homebrew-core/pull/63629.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Weirdly, linkage tests fail to highlight some of the issues: https://github.com/Homebrew/brew/issues/9321

**Comment by fxcoudert** [comment]
Need a revision bump, based on 10.14 results (and there may be some missing due to the brew issue in the previous link):

`boost chakra dwdiff freeling gjs hfstospell libcdr libical liblcf libmspub libphonenumber libpsl libvisio mpd ncmpcpp node node@10 node@12 node@14 openrct2 pazpar2 php php@7.2 php@7.3 sile tarantool tectonic tepl urweb vte3 vtk widelands yaz zebra znc zorba`

**Comment by derrabus** [comment]
`php@7.2` needs a patch additionally: #64044

**Comment by fxcoudert** [comment]
The brew issue is fixed, let's go for a 2nd round

**Comment by derrabus** [comment]
@fxcoudert No, my workaround just provides compatibility for icu4c 68.1. For Xcode 12, someone would need to backport https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0

File: Formula/php@7.2.rb [hunk 1]
```diff
@@ -97,6 +98,11 @@ def install
     # Prevent homebrew from harcoding path to sed shim in phpize script
     ENV["lt_cv_path_SED"] = "sed"
 
+    # Fix compilation with icu4c 68
```
Comment (SMillerDev) [inline Formula/php@7.2.rb:101]: Just disable it because it doesn't build.

**Comment by fxcoudert** [comment]
> For Xcode 12, someone would need to backport php/php-src@aa405b7

Oh that's an easy one, then. Done.

File: Formula/php@7.2.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/php@7.2.rb:101]: There's a known patch and it'll be deprecated in 3 days, so let's at least provide people with bottles :)

**Comment by miccal** [comment]
Thank you so much for your help with this @fxcoudert, it is most appreciated.

**Comment by fxcoudert** [comment]
There are some build issues:

```
brew install --verbose --build-bottle dwdiff
brew install --verbose --build-bottle libcdr
brew install --verbose --build-bottle libical
brew install --verbose --build-bottle urweb
brew install --verbose --build-bottle vtk
brew install --verbose --build-bottle zorba
```

There are also linkage test failures, some of which I don't understand…

**Comment by miccal** [comment]
`icu4c` has now been updated to 68.2.

@fxcoudert would you prefer me to close this PR and open a new one?

**Comment by fxcoudert** [comment]
@miccal whatever you prefer is fine with me

**Comment by miccal** [comment]
Sorry @fxcoudert, I cannot seem to rebase this locally, I am not sure why? After running `git rebase -i master` I get `noop` as the commit?

Are you able to update the `url`, `version` and `sha256`:
```
  url "https://github.com/unicode-org/icu/releases/download/release-68-2/icu4c-68_2-src.tgz"
  version "68.2"
  sha256 "c79193dee3907a2199b8296a93b52c5cb74332c26f3d167269487680d479d625"
```

**Comment by fxcoudert** [comment]
```
brew install --verbose --build-bottle gjs
brew install --verbose --build-bottle libcdr
brew install --verbose --build-bottle libical
brew install --verbose --build-bottle postgresql
brew install --verbose --build-bottle postgresql@10
brew install --verbose --build-bottle postgresql@11
brew install --verbose --build-bottle urweb
brew install --verbose --build-bottle zorba
```

**Comment by carlocab** [comment]
CI isn't currently in use, so it seems like a good time to get this going again. Rebased against master.

**Comment by carlocab** [comment]
I rebased incorrectly at some point and picked up a merge conflict for `php@7.4`. Whoops. 😞 

Also, `dartsim` will need an extra revision bump given https://github.com/Homebrew/homebrew-core/pull/68877

**Comment by miccal** [comment]
Thank you @carlocab.

**Comment by carlocab** [comment]
10.14
```
Error: 10 failed steps!
brew linkage --test mpd
brew test --retry --verbose mpd
brew install --verbose --build-bottle libcdr
brew install --verbose --build-bottle libical
brew install --verbose --build-bottle postgresql
brew install --verbose --build-bottle postgresql@10
brew install --verbose --build-bottle postgresql@11
brew install --verbose --build-bottle postgresql@12
brew install --verbose --build-bottle urweb
brew install --verbose --build-bottle zorba
```

10.15
```
Error: 13 failed steps!
brew fetch --retry minidlna
brew fetch --retry mpd
brew test --verbose couchdb
brew install --verbose --build-bottle libcdr
brew install --verbose --build-bottle libical
brew install --verbose --build-bottle postgresql
brew install --verbose --build-bottle postgresql@10
brew install --verbose --build-bottle postgresql@11
brew install --verbose --build-bottle postgresql@12
brew install --verbose --build-bottle urweb
brew audit znc --online --git --skip-style
brew install --verbose --build-bottle zorba
brew audit zorba --online --git --skip-style
```

11.1
```
Error: 21 failed steps!
brew fetch --retry minidlna
brew fetch --retry mpd
brew test --verbose couchdb
brew install --verbose --build-bottle dartsim
brew install --verbose --build-bottle gjs
brew install --verbose --build-bottle libcdr
brew install --verbose --build-bottle libical
brew install --verbose --build-bottle node@14
brew install --verbose --build-bottle postgresql
brew install --verbose --build-bottle postgresql@10
brew install --verbose --build-bottle postgresql@11
brew install --verbose --build-bottle postgresql@12
brew audit tarantool --online --git --skip-style
brew audit tectonic --online --git --skip-style
brew install --verbose --build-bottle urweb
brew audit urweb --online --git --skip-style
brew audit vtk --online --git --skip-style
brew audit wangle --online --git --skip-style
brew audit znc --online --git --skip-style
brew install --verbose --build-bottle zorba
brew audit zorba --online --git --skip-style
```

**Pull request closed without merging**